package/test/beamer_delegate_test.dart

```diff
@@ -402,13 +402,19 @@ void main() {
       rootDelegate.beamToNamed('/test'); // initial will update
```

Comment:
we should also remove this "// initial will update" comment which is now wrong after your fix.

Reply:
@samdogg7 you forgot to remove this comment in your last commit :smiley:

Reply:
Oops, I removed it from the wrong test case. Updated now.